src/pybind/mgr/dashboard/frontend/src/app/shared/components/grafana/grafana.component.ts

```diff
@@ -23,10 +23,43 @@ export class GrafanaComponent implements OnInit, OnChanges {
   panelStyle: any;
   grafanaExist = false;
   mode = '&kiosk';
-  modeFlag = false;
-  modeText = 'Change time selection';
   loading = true;
   styles = {};
+  time: string;
+  grafanaTimes = {
```

Comment:
Why not:
- **Time**: now, yesterday, 1 week ago, etc.
- **Offset/look-back**: +-1 hour, +-1day, +-1week, etc.

This way we provide more possible combinations with less amount of entries.